versions/app/src/screens/referScreen.js

```diff
@@ -5,53 +5,80 @@ import React, { useState } from 'react'
 import { SafeAreaView } from 'react-native-safe-area-context';
 
 // COMPONENTS
-import { Text, StyleSheet, View } from 'react-native'
+import { Text, StyleSheet, View, Alert } from 'react-native'
+import { CustomButton, SubmitButton, ReferralButton } from '../components/Button'
 
 // STYLES
-import {globalStyles, TouchableHighlight, footer, forms} from '../styles/styles';
+import { globalStyles, TouchableHighlight, footer, forms } from '../styles/styles';
 import Theme from '../styles/theme.style.js';
+import { MaterialIcons, Feather, Fontisto, FontAwesome } from '@expo/vector-icons';
 
 /*******************************************************************************/
 // MAIN EXPORT FUNCTION
 /*******************************************************************************/
 
 
-export default function ReferScreen({navigation}) {
+export default function ReferScreen({ navigation }) {
+
+    const buttonPress = () => {
```

Comment:
Is this used?

Reply:
This was my test to see if I could knock out some tasks as I was being onboarded. The screen was deprioritized for right now in favor to get the core features out of the MVP.